py3-graph-tool.yaml

```diff
@@ -78,15 +82,17 @@ subpackages:
       - uses: autoconf/configure
         with:
           opts: |
-            --enable-openmp \
             --includedir=/usr/lib/cairomm-1.16 \
-            --disable-debug \
-            --disable-dependency-tracking \
             PYTHON=python${{range.key}}
       - runs: |
-          make -C . -j$(nproc) V=1
+          # Only link when needed
+          sed -i -e 's/ -shared / -Wl,-O1,--as-needed\0/g' libtool
+
+          # Unfortunately cuts the number of threads used in
+          # half, but we need lots of memory per job
+          make -C . -j$(( $(nproc) / 2 ))
       - uses: autoconf/make-install
-      - runs: find ${{targets.contextdir}} -name "*.so" -exec strip -g {} \;
+      - uses: strip
```

Comment:
This one I think we'll want to double-check. I think there was an issue with scanelf and strip being too conservative and the final package ended up being multiple GB larger than others.